manticore/platforms/decree.py

```diff
@@ -10,6 +10,7 @@
 from ..utils.helpers import issymbolic
 from ..binary import CGCElf
 from ..platforms.platform import Platform
+from ..platforms.platform import RestartSyscall, Deadlock, BadFd, SyscallNotImplemented
```

Comment:
So the idea is that all(or some) the platform exceptions end up at platorm.py and then that's managed by carefully importing what we need in the different places? 
I know that some of them will not make sense for other platforms and non should be used upstream or in State or Executor.